base/PerfOptions.php

```diff
@@ -291,6 +295,26 @@ public function __construct(Vector<string> $argv) {
     $this->daemonOutputToFile = $this->getBool('daemon-files');
 
     $argTempDir = $this->getNullableString('temp-dir');
+    
+    if(array_key_exists('hhvm-server-threads', $o)){
+      $this->hhvmServerThreads = $this->getNullableString('hhvm-server-threads');
+    } else {
+      $this->hhvmServerThreads = '100';
+    }
+    $hhvmDir  = realpath(__DIR__ . '/..').'/conf/php.ini';
+    $file_contents = file_get_contents($hhvmDir);
+    $file_contents = preg_replace('/^.*hhvm.server.thread_count.*$/m', 'hhvm.server.thread_count='.$this->hhvmServerThreads, $file_contents );
+    file_put_contents($hhvmDir, $file_contents);
+
+    if(array_key_exists('benchmark-concurrency', $o)){
+      $this->benchmarkConcurrency = $this->getNullableString('benchmark-concurrency');
+    } else {
+      $this->benchmarkConcurrency = '200';
+    }
+    $bmConcurrency = __DIR__.'/PerfSettings.php';
```

Comment:
Nope, don't modify this file.
 - nothing that modifies 'git status' is OK
 - moving it to `PerfOptions` is the correct thing to do.